provision/aws/rosa_install_ispn_operator.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+INSTALL_NAMESPACE=${INSTALL_NAMESPACE:-openshift-operators}
+
+# Create subscription for Infinispan Operator
+oc apply -f - << EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: infinispan
+  namespace: ${INSTALL_NAMESPACE}
+spec:
+  channel: 2.3.x
+  installPlanApproval: Manual
+  name: infinispan
+  source: community-operators
+  sourceNamespace: openshift-marketplace
```

Comment:
If we set both `source` and `sourceNamespace` via env variables this will help reuse between minikube/openshift.